events/class_tests/event_tests.py

```diff
@@ -90,7 +90,7 @@ def test_get_all_user_month_meetings(self, event_title, user_id, year, month):
 
     @pytest.mark.parametrize('event_title, user_id, date', [
         ("event1", 1, datetime(2022, 3, 24)),
-        ("event3", 1, datetime(2022, 1, 24))
+        ("event3", 1, datetime(2023, 1, 24))
     ], ids=[
         "get all users day meeting of event 1",
         "get all users month meeting of event 3"
```

Comment:
can you please change 'month' into 'day'

Reply:
done